docs/dApps/quickstart.md

```diff
@@ -1,5 +1,405 @@
-# dApp quickstart
```

Comment:
Globally, I have the impression that this quickstart dapp is very similar, and roughly the same order of complexity as the one in "Build your first dapp" (in https://docs-staging-git-dapp-quickstart-trili-tech.vercel.app/tutorials/build-your-first-app). Is it worth introducing this new howto? Maybe it would be better and simpler to leave the introducing docs in the docs section, and link for a quickstart directly to the other one?

Reply:
Yes, after leaving this PR sit for a while I think it may not be worth duplicating the content in the tutorials. This quickstart just provided a tutorial with as little explanation as possible. Might not be worth the cost of adding and maintaining it.

Reply:
I see the initial motivation, but the result shows one cannot significantly reduce the amount of details. And it also blurs the distinction between the doc part and the tutorials part on the website.

So yes, let's drop it.